### ISSUE TITLE
Firefox hardware acceleration causing specific window screen not to share

### ISSUE THREAD

**Issue by thoughtfuldata** [jitsi/jitsi-meet#4462]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed.
General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Description
---
Firefox 68.0.1 and 68.0.0 hardware acceleration causing specific window screen not to share. it works if you share the entire page or other programs (except for other browsers like chrome).
## Current behavior
---
Basically when screen sharing with hardware acceleration turned on it wont screen share. it just shows the window name and a blank page
## Expected Behavior
---
to show screen
## Possible Solution
---
must be something with hardware acceleration inside FireFox. I got it to work after turning off hardware acceleration.
## Steps to reproduce
---
nothing special to reproduce
![for meet jit si](https://user-images.githubusercontent.com/11638304/61511753-91f91500-a9ac-11e9-904a-ed27d5e129d4.JPG)

# Environment details
---
Firefox 68.0.1 and 68.0.0

**Comment by thoughtfuldata** [comment]
After turning hardware acceleration back on, it works again. May have reset something when turning it back on and off

**Comment by saghul** [comment]
There is nothing we can do on the application side to mitigate this. If the issue persists / resurfaces, please report it to Firefox, as it looks like a bug in the browser itself.
